### ISSUE TITLE
atomic trust default adds entry into docker transport

### ISSUE THREAD

**Issue by ddarrah** [projectatomic/atomic#626]
If there is no default section in the policy.json, using the 'atomic trust default' command adds it to the docker transport.  It edits the policy fine if it is in the file.  This is probably low priority, just being thorough.

```
[root@rhel-base openshift-origin-client-tools-v1.3.0-rc1-ac0bb1bf6a629e0c262f04636b8cf2916b16098c-linux-32bit]# atomic trust default accept
'default'
[root@rhel-base openshift-origin-client-tools-v1.3.0-rc1-ac0bb1bf6a629e0c262f04636b8cf2916b16098c-linux-32bit]# cat /etc/containers/policy.json 
{
    "transports": {
        "docker": {
            "default": [
                {
                    "type": "insecureAcceptAnything"
                }
            ]
        }, 
        "atomic": {}, 
        "dir": {
            "registry.example.com/something": [
                {
                    "keyType": "GPGKeys", 
                    "type": "signedBy", 
                    "keyPath": "/etc/containers/pki/ddarrah@redhat.com.key"
                }
            ]
        }
    }
```

**Comment by rhatdan** [comment]
@aweiteka PTAL

**Comment by aweiteka** [comment]
@ddarrah this should be fixed by #630
